mediated_transfer.rst

```diff
@@ -0,0 +1,71 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+The protocol supports mediated transfers. Nodes may use them to send payments.
+A :term:`Mediated transfer` may be cancelled and can expire until the initiator reveals the secret.
+
+A mediated transfer is done in two stages, possibly on a series of channels:
+
+- Reserve token :term:`capacity` for a given payment, using a :ref:`locked transfer message <locked-transfer-message>`.
```

Comment:
Can we put 
- **Allocation:** reserves token ...
- **Finalization:** use the reserved token amount either to unlock or ...